### PR TITLE
test(tooltip): add testcase

### PR DESCRIPTION
**Pull request by zenml**
## PR Checklist

- [x] Label has been added

- [x] Fix linting errors

## Change information

### CODE REVIEW

**Comment by unix** [comment]
@zenml  Thanks for your contribution, when you are ready for review, let me know.

File: components/tooltip/__test__/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import React from 'react';
```
Comment (unix) [inline components/tooltip/__test__/index.test.tsx:1]: no semicolon required

File: components/tooltip/__test__/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  expect(wrapper.find('.inner').length).toBe(0);
+};
+
+describe('Tooltip', () => {
+  it('should render correctly', async () => {
+    const myThme = {
+      type: 'dark',
+    } as ThemeParam;
+    const wrapper = mount(
+      <ZEITUIProvider theme={myThme}>
```
Comment (unix) [inline components/tooltip/__test__/index.test.tsx:23]:
```suggestion
      <ZEITUIProvider theme={{ type: 'dark' }}>
```

File: components/tooltip/__test__/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    );
+
+    await updateWrapper(wrapper, 150);
+    expect(wrapper.find('#visible').length).toBe(1);
+  });
+
+  it('should render correctly by using wrong placement', async () => {
+    const wrapper = mount(
+      <div>
+        <Tooltip text={<p id='initialVisible'>custom-content</p>} initialVisible={true} placement={'test' as any}>
```
Comment (unix) [inline components/tooltip/__test__/index.test.tsx:109]:
```suggestion
        <Tooltip text={<p id='initial-visible'>custom-content</p>} initialVisible={true} placement={'test' as any}>
```

File: components/tooltip/__test__/index.test.tsx (hunk 2 above)
Comment (zenml) [reply components/tooltip/__test__/index.test.tsx:23]: > 
这样会类型不兼容吧

**Pull request merged**